Task_DEV_6/Task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+namespace Task_DEV_6
+{
+    class CustomStack
+    {
+        private const int LENGTH = 10;
+        private const string EXCEPTION_MESSAGE = "There are no elements on the stack.";
```

Comment:
It is better to specify in the name because of what the error occurred.

Reply:
Done.